app/models/kpi.rb

```diff
@@ -0,0 +1,366 @@
+class Kpi < CouchRest::Model::Base
+  include Record
+
+  INTRODUCTION_AND_ENGAGEMENT_TYPE = {
+    name: 'introduction_and_engagement',
+    calculations: ['rate_of_report', 'elapsed_time_for_incident_and_report', 'average_delay_of_access'],
+    days_ranges: [
+      [0, 3],
+      [4, 5],
+      [6, 14],
+      [15, 30],
+      [31, 92]
+    ]
+  }
+
+  ASSESSMENT_TYPE = {
+    name: 'assessment',
+    calculations: ['completed_asssessment', 'approved_assessment']
+  }	
+    CASE_ACTION_PLANNING_TYPE = {
```

Comment:
Indentation of CASE_ACTION_PLANNING_TYPE should align with ASSESSMENT_TYPE above